tree_traversal_without recursion.py

```diff
@@ -0,0 +1,63 @@
+class stack:
+    def __init__(self):
+        self.stack = []
+    def push(self, item):
+        self.stack.append(item)
+    def peek(self):
+        if self.isEmpty():
+            return None
+        else:
+            return self.stack[-1]
+    def pop(self):
+        if self.isEmpty():
+            return None
+        else:
+            return self.stack.pop()
+    def size(self):
+        return len(self.stack)
+    def isEmpty(self):
+        return self.size() == 0
+
+class Node:
+    def __init__(self, data):
+        self.data = data
+        self.left= None
+        self.right = None
+        
+        
+#find max_height of a tree
+def in_order_traversal(root):
+    curr = root
+    stack = stack()
+    while True:
+        while curr is not None:
+            stack.push(curr)
+            curr = curr.left
+        if stack.size() == 0:
+            break
+        curr = stack.pop()
+        print(curr.data)
+        curr =curr.right

```

Comment:
Space before and after '=' for consistency. Use `pylint` to validate your files in the future.